src/components/FooterCheckout/FooterCheckout.tsx

```diff
@@ -8,7 +8,7 @@ import { LogoPagoPACompany } from "@assets/LogoPagoPACompany";
 type FooterCheckoutProps = LangSwitchProps & {
   companyLink: CompanyLinkType;
   links: Array<FooterLinksType>;
-  onExit?: (href: string, linkType: LinkType) => void;
+  onExit?: (linkType: LinkType, href?: string) => void;
```

Comment:
it's not recommended to change the signature of a method inside a Prop, you risk to break all the application that are using it!